src/MSBuild.UnitTests/TerminalLogger_Tests.cs

```diff
@@ -52,6 +52,9 @@ public TerminalLogger_Tests()
             _terminallogger.CreateStopwatch = () => new MockStopwatch();
 
             UseProjectRelativeDirectory("Snapshots");
+
+            // Avoids issues with different cultures on different machines
+            Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
```

Comment:
Can you reset this when done with this test class? I don't want it to leak to other tests.